spec/factories/event_factory.rb

```diff
@@ -2,8 +2,8 @@
   factory :event do
     description "123"
     user "user"
-    location "location"
     start_time Time.now
     end_time Time.now + 10.minutes
+    conference_room
```

Comment:
this should be an association:
`association : conference_room, factory: :conference_room `